android-components/components/feature/fxsuggest/src/main/java/mozilla/components/feature/fxsuggest/FxSuggestStorage.kt

```diff
@@ -22,10 +23,9 @@ import mozilla.appservices.suggest.SuggestionQuery
 class FxSuggestStorage(
     context: Context,
 ) {
-    // Lazily initializes the store on first use. `Context.getDatabasePath()` does I/O, so
-    // `store.value` should only be accessed from the read or write scope.
+    // Lazily initializes the store on first use.
```

Comment:
Breadcrumb for our future selves: accessing `cacheDir` and using the `File` constructor also does blocking I/O.